.github/PROCESS.md

```diff
@@ -16,14 +16,35 @@ Keep guides short: we don't create rules; we automate.
 2. Pull request **title** should be accurate and descriptive as it is used in the generated changelog.
    It should not end with a dot.
    It should not mention the issue number but describe the fixed issue instead.
-3. Pull request **description** typically just mentions the issue number like "Refs #123." or "Closes #456.".
+3. Pull request **description** must mention the issue number like "Refs #123." or "Closes #456.".
    Do not use `https://github.com/org/repo/issue/123` even if it is rendered the same on GitHub.
+   We provide a pull requests template that includes suggestions and readiness checklist.
    End sentences with a dot.
 4. Commit messages (both titles and bodies) are not important and can be "WIP" or anything else.
    All commits are always squashed on merge by GitHub, so there is no need to squash them manually, amend them, and/or do force pushes.
-   Autogenerated GitHub's squash commit's title and body should be manually replaced by PR's title and description; that includes both manual merges and auto-merges.
+   Autogenerated GitHub's squash commit's title should be manually replaced by PR's title.
+   Autogenerated GitHub's squash commit's body should be manually replaced by "Refs #{issue_number}". 
    Do not leave original commit messages in the squashed commit body.
 
+## Draft pull requests
+
+1. We aim our issues to be scoped and well-described. 
+   With that, ideally, there shouldn't be a need to create a draft pull request.
+2. We aim to implement and merge improvements as fast as possible to continuously increase 
+   [Lead Time for Changes](https://cloud.google.com/blog/products/devops-sre/using-the-four-keys-to-measure-your-devops-performance). 
+   Having too many drafts indicates that something is wrong in our processes, it needs to be discussed and changed.
+3. Rarely, there is a need to demonstrate something that shouldn't be merged. 
+   For example, a corner case that reveals a bug in the code that can't be fixed easily.
+   For such cases, draft pull requests are possible.
```

Comment:
It's not clear:
* if I create a draft PR with the test case for the bug? 
* If I create a PR with my functionality - so what is it for then?

Reply:
Yeah, let me remove this case, so the whole picture is less confusing.